content/efcore/getting-started/samples/loading-related-data-tutorial/include-and-then-include-method-chain/Program.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var book = context.Books
```

Comment:
I'm fine with using it even though it only has one edition - however, we either need to change it so that it is `== "The Call of the Wild"` or `.Contains("Call of the Wild")`, because right now it is failing without the `The`